src/ensembl/src/content/app/species/state/general/speciesGeneralHelper.ts

```diff
@@ -0,0 +1,419 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as urlFor from 'src/shared/helpers/urlHelper';
+import { getCommaSeparatedNumber } from 'src/shared/helpers/formatters/numberFormatter';
+
+import { SpeciesStatsProps } from 'src/content/app/species/components/species-stats/SpeciesStats';
+import { ExampleFocusObject } from 'src/shared/state/genome/genomeTypes';
+
+import { sampleData } from '../../sample-data';
+import { buildFocusIdForUrl } from 'src/shared/state/ens-object/ensObjectHelpers';
+import { urlObj } from 'src/shared/components/view-in-app/ViewInApp';
+
+export enum SpeciesStatsSections {
```

Comment:
1) Enums are usually named in singular, because they represent not a set of things, but one thing belonging to a set. See [example in typescript docs](https://www.typescriptlang.org/docs/handbook/enums.html#numeric-enums).

2) Aren't we using enums a bit too much? What would this code look like if we didn't use enums?

Reply:
> Aren't we using enums a bit too much? What would this code look like if we didn't use enums?
Any suggestions for alternatives?